torch/ao/quantization/fx/prepare.py

```diff
@@ -598,41 +658,53 @@ def _maybe_insert_input_observer_for_arg_or_kwarg(
         # regular flow for most nodes, except standalone modules
         is_weight = node_arg_is_weight(node, arg)
 
+        # TODO: we are assuming "target_dtype_info" exists here, maybe
+        # a default value also need to be provided here
+        target_dtype_info = node.meta["target_dtype_info"]
         # for nodes that doesn't have `reuse_input_obs_or_fq` configured,
         # we'll default to False, this makes configuring this field optional for users
-        reuse_input_obs_or_fq = node.meta["target_dtype_info"].get("reuse_input_obs_or_fq", False)
+        reuse_input_obs_or_fq = target_dtype_info.get("reuse_input_obs_or_fq", False)
 
-        act_post_process_ctr = node.meta["target_dtype_info"]["weight_obs_or_fq_ctr"] if is_weight else \
-            node.meta["target_dtype_info"]["input_act_obs_or_fq_ctr"]
+        # TODO: check bias as well?
+        obs_or_fq_ctr_key = "weight_obs_or_fq_ctr" if is_weight else "input_act_obs_or_fq_ctr"
+        act_post_process_ctr = target_dtype_info.get(obs_or_fq_ctr_key, _DEFAULT_FP32_OBS_OR_FQ_CTR)
 
         arg_as_output_target_dtype = _get_arg_target_dtype_as_output(arg, named_modules)
+        arg_as_output_target_is_dynamic = _get_arg_target_is_dynamic_as_output(arg, named_modules)
```

Comment:
So real special case here is lstm where we need to trace back through dequant stub to find the node? Is this to preserve existing code for fx quant? and will this be relevant for new workflow?

Reply:
yeah not relevant for new flow, that code path is not used